README.md

```diff
@@ -1,56 +1,57 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Budget app
 
 > Description the project.
-
+> 
+This is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Ruby On Rails
+- Github
+- Devise
+- Css
+- Bootstrap
 
-## Live Demo (if available)
+## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://secret-waters-08899.herokuapp.com/)
 
 
+## Video Demo
+[ Video Demo ](https://www.loom.com/share/2cc0457ff83241f481e19dcf99bab6cd)
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
 
 
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
-
+Ruby >= 3.0.0
 ### Setup
+Run git clone on this project at the desired directory:
 
+https://github.com/peterdgreat/Budget-App
 ### Install
-
+bundle install
 ### Usage
 
```

Comment:
- [ ] I would recommend including all the steps to run the project locally so that any user or recruiter can easily run the project.

```
git clone https://github.com/peterdgreat/Budget-App.git
cd Budget-App.

bundle
rails db:create
rails db:migrate
rails db:seed // if you have any seed data
rails s
```